Makefile

```diff
@@ -120,19 +122,26 @@ endif
 .PHONY: release
 release: build archive
 
+.PHONY: test-deps
+test-deps:
+	# The desire here is to download this test dependency without polluting go.mod
+	# In golang >=1.16 there is a new way to do this with `go install gotest.tools/gotestsum@latest`
```

Comment:
The CLI is already changed to Go 1.16. Why do we have to do it the old way for now? The workflow runs with Go 1.16 so `go install` should work?

Reply:
It will, I didn't want to force our contributors to upgrade, especially since we haven't even done a release with 1.16 yet.

I can change it, if you think it is necessary. I just want to merge this PR so we can start collecting data.

Reply:
If the old way works for both, I am fine with that.